reana_server/config.py

```diff
@@ -301,11 +301,11 @@ def _get_rate_limit(env_variable: str, default: str) -> str:
 # Workspace retention rules
 # ==================
 
-MAXIMUM_WORKSPACE_RETENTION_PERIOD = 365  # TODO: specify this through Helm
+WORKSPACE_RETENTION_PERIOD = int(os.getenv("WORKSPACE_RETENTION_PERIOD", "365"))
```

Comment:
I think we should move all the changes related to the renaming of this variable to the first commit (both in this file and in `utils.py`)